license-format.xml

```diff
@@ -10,7 +10,7 @@
   </java_style>
   <custom_xml_style>
     <firstLine><![CDATA[<!--]]></firstLine>
-    <beforeEachLine> </beforeEachLine>
+    <beforeEachLine>//</beforeEachLine>
```

Comment:
Was this change intentional?

Reply:
@bashir2 this was intended because the[ xml-formatter](https://acegi.github.io/xml-format-maven-plugin/usage.html) doesnt accept empty tags with this [error ](https://pastebin.com/dNPuM6xc) ,, thats why in introduced ` // `in the `<custom_xml_style> ` of the` license.format-xml `hence the C/Java style comment symbols appearing in the XML file

Reply:
Then maybe this is not a good formatter to use? These are multi-line XML comments which are valid XML so I expect a formatter to accept them (without `//`).